recipes/openpathsampling/meta.yaml

```
@@ -0,0 +1,61 @@
+{% set name = "openpathsampling" %}
+{% set version = "1.1.0" %}
+{% set sha256 = "81c2a11e4bf6c6ba7a4a8ae47ce28bc6eb7a2dfe9b74169ed77fe9515e10a782" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  # note to self: if we add win support, switch skip line to noarch: python
+  skip: True  # [win]
+  script: "{{ PYTHON }} -m pip install . -vv"
```

```suggestion
  script: "MPLBACKEND=Agg {{ PYTHON }} -m pip install . -vv"
```

But I suspect that this is going to break the windows builds?

---

This particular recipe skips Windows, so that shouldn't be a problem in this specific case. (I suspect we'll drop Python 2.7 support before we get around to supporting Windows.)

---

Ah, this doesn't fix because this change affects the installation; the failure happens on the `import` testing. I would need to add the environment variable to the invocation of `conda-build`. I don't know how I would go about doing that....